test/migrations/variables.hrx

```diff
@@ -0,0 +1,63 @@
+<==> recursive_manifest
+one.scss -> two.scss three.scss
+two.scss -> three.scss
+four.scss -> three.scss
+
+<==> input/one.scss
+@import "two";
+span {
+  color: $a;
+  background: $b;
+  height: $c;
+}
+
+<==> input/two.scss
+@import "three";
+$a: green;
+a {
+  $c: red;
+  background: $b;
+  color: $c;
+}
+
+<==> input/three.scss
+$b: blue;
+$c: 5px !default;
+b {
+  width: 0;
+}
+
+<==> input/four.scss
+$c: 4px;
+@import "three";
+
+<==> expected/one.scss
+@use "three";
+@use "two";
+span {
+  color: $two.a;
+  background: $three.b;
+  height: $three.c;
+}
+
+<==> expected/two.scss
+@use "three";
+$a: green;
+a {
+  $c: red;
+  background: $three.b;
+  color: $c;
+}
+
+<==> expected/three.scss
+$b: blue;
+$c: 5px !default;
+b {
+  width: 0;
+}
+
+<==> expected/four.scss
+$c: 4px;
```

Comment:
At some point, it would be good to remove variable declarations that are used to configure an import. Maybe file an issue for that?